frontend/public/components/factory/list.tsx

```diff
@@ -294,7 +295,7 @@ const VirtualRows: React.SFC<RowsProps> = (props) => {
       parent={parent}
       rowIndex={index}>
       <div style={style} className="co-m-row">
-        <Row key={_.get(obj, 'metadata.uid', index)} obj={obj} expand={expand} kindObj={kindObj} index={index} />
+        <Row key={_.uniqueId(`${uid}--`)} obj={obj} expand={expand} kindObj={kindObj} index={index} />
```

Comment:
This will cause the DOM nodes to be recreated every time even if something hasn't changed.